astropy/cosmology/flrw/base.py

```diff
@@ -1162,7 +1162,7 @@ def critical_density(self, z):
 
         Returns
         -------
-        rho : `~astropy.units.Quantity`
+        rho : Quantity
             Critical density in g/cm^3 at each input redshift.
```

Comment:
While you are at it, maybe remove "in g/cm^3" here? From the code, it depends on the units of `critical_density0`, but in any case should be irrelevant as long as the unit is equivalent. But of course then it would be logical to also remove the unit in all the other places too, so feel free to ignore.